docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -7,4 +12,342 @@
+
+$$ L1 \sim P(100,0.3) $$
+$$ L2 \sim P(100,0.5) $$
+$$ L3 \sim P(100,0.2) $$
+
+![](../../../img/EVPoisson.png)
+
+What if we weren't entirely certain that the average values for $L1, L2, L3$ are $0.3, 0.5, 0.2$, we can also sample the averages from a normal distribution centered on these values with reasonable standard deviations. We can say that:
+
+$$ \lambda \sim N(\mu,\sigma) $$
+
+Which means that the input to $L$ is distributed normally with average $\mu$ and standard deviation $\sigma$.
+
+Our final distribution for modeling the anticipated need for each level of charging port in our $k = 100$ EV garage can be written as:
+
+$$ L \sim P(k,\lambda) $$
+$$ \lambda \sim N(\mu,\sigma)  $$
+
+<center>
+
+|          | $L1$          |     $L2$      |                $L3$ |
+| -------- | :------------ | :-----------: | ------------------: |
+| $\mu$    | 0.3           |      0.5      |                 0.2 |
+| $\sigma$ | $\sim N(1,3)$ | $\sim N(1,2)$ | $\sim N(0.05,0.25)$ |
+
+</center>
+
+![](../../../img/EVfulldist.png)
+
+Notice that the individual overplotted distributions in the two histograms above are different -- there is more flexibility encoded into the second. The distributions in the second plot describe the following assumptions about the anticipated need for Level 1, 2, and 3 chargers:
+
+1. The number of charging ports is discrete and positive (Poisson).
+2. The variance in the number of ports should be normally distributed.
+3. The average percentage of Level 1, 2, and 3 chargers is 30%, 50%, and 20%, respectively.
+4. The variance around the average is uniformly distributed.
+5. The variance is relatively very broad for Level 1, broad for Level 2, and very constrained for Level 3. This means we are very confident in our guess of the average percentage for Level 3 chargers, less confident for Level 2, and much less confident for Level 3.
+
+We have described the individual distributions for each level of charging port. What we don't know, prior to running the Monte Carlo co-simulation, is how these distributions will jointly impact the research question.
+
+#### Research Question Configuration
+
+We want to address the research question: What is the likely power draw that EVs will demand?
+
+At the beginning of the co-simulation, the distributions defined above will be sampled $N$ times within the EV federate, where $N =$ the number of parking spots/charging ports in the garage. The output of the initial sampling is the number of requested Level 1, 2, and 3 charging ports. The soc for the batteries on board are initialized to a uniform random number between 0.05 and 0.5, and these SOC are sent to the Charger federate. If the SOC of an EV battery is less than 0.9, the EV Controller federate tells the EV battery in the EV federate to continue charging. Otherwise, the EV Charger disconnects the EV battery, and instructs the it to sample a new EV battery from the distributions (one sample).
```

Comment:
```suggestion
At the beginning of the co-simulation, the distributions defined above will be sampled $N$ times within the EV federate, where $N =$ the number of parking spots/charging ports in the garage. The output of the initial sampling is the number of requested Level 1, 2, and 3 charging ports. The SOC for the batteries on board are initialized to a uniform random number between 0.05 and 0.5, and these SOC are sent to the Charger federate. If the SOC of an EV battery is less than 0.9, the EV Controller federate tells the EV battery in the EV federate to continue charging. Otherwise, the EV Charger disconnects the EV battery, and instructs the it to sample a new EV battery from the distributions (one sample).
```